RuntimeException related to CachePopup.onCreate

This problem report from Google Play has the highest occurence (besides the NPE on Resume issues, which are hopefully already fixed).

Version: 2019.02.23 (and also beta 2019.03.30-RC1)
```
java.lang.RuntimeException: 
 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2822)
 
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:2897)
 
  at android.app.ActivityThread.-wrap11 (Unknown Source)
 
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1598)
 
  at android.os.Handler.dispatchMessage (Handler.java:105)
 
  at android.os.Looper.loop (Looper.java:251)
 
  at android.app.ActivityThread.main (ActivityThread.java:6572)
 
  at java.lang.reflect.Method.invoke (Native Method)
 
  at com.android.internal.os.Zygote$MethodAndArgsCaller.run (Zygote.java:240)
 
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:767)
Caused by: android.util.AndroidRuntimeException: 
 
  at android.support.v7.app.AppCompatDelegateImplV7.throwFeatureRequestIfSubDecorInstalled (AppCompatDelegateImplV7.java:1683)
 
  at android.support.v7.app.AppCompatDelegateImplV7.requestWindowFeature (AppCompatDelegateImplV7.java:585)
 
  at android.support.v7.app.AppCompatActivity.supportRequestWindowFeature (AppCompatActivity.java:228)
 
  at cgeo.geocaching.activity.AbstractActivity.onCreateCommon (AbstractActivity.java:134)
 
  at cgeo.geocaching.activity.AbstractActivity.onCreate (AbstractActivity.java:114)
 
  at cgeo.geocaching.CachePopup.onCreate (CachePopup.java:40)
 
  at android.app.Activity.performCreate (Activity.java:6975)
 
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1214)
 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2775)
```

Recently @rsudev implemented a mitigation for the above mentioned exception. We have this code in our beta version 2019.08.01.-RC.
Checking the Google Play Store I can see, that there is still this exception but now marked as *new* as the triggering code lines changed. Maybe @rsudev or someone else can get a clue from this:
```
java.lang.RuntimeException: 
 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3122)
 
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3261)
 
  at android.app.servertransaction.LaunchActivityItem.execute (LaunchActivityItem.java:78)
 
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:108)
 
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:68)
 
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1977)
 
  at android.os.Handler.dispatchMessage (Handler.java:106)
 
  at android.os.Looper.loop (Looper.java:193)
 
  at android.app.ActivityThread.main (ActivityThread.java:6923)
 
  at java.lang.reflect.Method.invoke (Native Method)
 
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:493)
 
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:870)
Caused by: android.util.AndroidRuntimeException: 
 
  at android.support.v7.app.AppCompatDelegateImplV9.throwFeatureRequestIfSubDecorInstalled (AppCompatDelegateImplV9.java:1731)
 
  at android.support.v7.app.AppCompatDelegateImplV9.requestWindowFeature (AppCompatDelegateImplV9.java:605)
 
  at android.support.v7.app.AppCompatActivity.supportRequestWindowFeature (AppCompatActivity.java:234)
 
  at cgeo.geocaching.CachePopup.onCreate (CachePopup.java:41)
 
  at android.app.Activity.performCreate (Activity.java:7148)
 
  at android.app.Activity.performCreate (Activity.java:7139)
 
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1293)
 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3102)
```

```
java.lang.RuntimeException: 
 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3047)
 
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3182)
 
  at android.app.ActivityThread.handleRelaunchActivityInner (ActivityThread.java:4930)
 
  at android.app.ActivityThread.handleRelaunchActivity (ActivityThread.java:4839)
 
  at android.app.servertransaction.ActivityRelaunchItem.execute (ActivityRelaunchItem.java:69)
 
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:108)
 
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:68)
 
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1916)
 
  at android.os.Handler.dispatchMessage (Handler.java:106)
 
  at android.os.Looper.loop (Looper.java:193)
 
  at android.app.ActivityThread.main (ActivityThread.java:6898)
 
  at java.lang.reflect.Method.invoke (Native Method)
 
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:537)
 
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:858)
Caused by: android.util.AndroidRuntimeException: 
 
  at android.support.v7.app.AppCompatDelegateImplV9.throwFeatureRequestIfSubDecorInstalled (AppCompatDelegateImplV9.java:1731)
 
  at android.support.v7.app.AppCompatDelegateImplV9.requestWindowFeature (AppCompatDelegateImplV9.java:605)
 
  at android.support.v7.app.AppCompatActivity.supportRequestWindowFeature (AppCompatActivity.java:234)
 
  at cgeo.geocaching.CachePopup.onCreate (CachePopup.java:41)
 
  at android.app.Activity.performCreate (Activity.java:7149)
 
  at android.app.Activity.performCreate (Activity.java:7140)
 
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1288)
 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3027)
```

--- a/main/src/cgeo/geocaching/CachePopup.java
+++ b/main/src/cgeo/geocaching/CachePopup.java
@@ -2,6 +2,7 @@ package cgeo.geocaching;
 
 import cgeo.geocaching.activity.AbstractActivity;
 import cgeo.geocaching.activity.ActivityMixin;
+import cgeo.geocaching.utils.Log;
 
 import android.app.Activity;
 import android.content.Context;
@@ -10,6 +11,7 @@ import android.os.Bundle;
 import android.support.v4.app.DialogFragment;
 import android.support.v4.app.Fragment;
 import android.support.v4.app.FragmentTransaction;
+import android.util.AndroidRuntimeException;
 import android.view.Window;
 
 import org.apache.commons.lang3.StringUtils;
@@ -38,7 +40,11 @@ public class CachePopup extends AbstractActivity {
     @Override
     public void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        supportRequestWindowFeature(Window.FEATURE_NO_TITLE);
+        try {
+            supportRequestWindowFeature(Window.FEATURE_NO_TITLE);
+        } catch (final AndroidRuntimeException ex) {
+            Log.e("Error requesting no title feature", ex);
+        }
         this.setTheme(ActivityMixin.getDialogTheme());
 
 

